.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+# Summary
+
+## Issue
+
+<!-- Some issue -->
+
+## Changes
+
+<!-- Ipsum lorem. -->
+
+## Specific Testing (Optional)
+
+<!-- Any specific text used to test outside of typical Ruff workflows -->
```

Comment:
## Documentation
<!-- If the PR introduces changes that need updating to the documentation, describe those changes here -->